ios-base/DroidKaigi 2020/SideBar/Views/SidebarViewController.swift

```diff
@@ -0,0 +1,61 @@
+import Material
+import UIKit
+
+final class SidebarViewController: UITableViewController {
```

Comment:
nits: The directory name `SideBar` and the class prefix `Sidebar` are do not match, I think better to match them.

Reply:
Exactly, I missed! I will fix. Thanks! 👍